<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-J8JF1UBS" failures="1" tests="1" name="org.example.forcegetregister.runners.TestRunner" time="139.601" errors="0" timestamp="2025-03-01T13:44:30 EET" skipped="0">
  <testcase classname="org.example.forcegetregister.runners.TestRunner" name="runScenario" time="139.601">
    <failure message="Login was not successful - Welcome message not found! expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Login was not successful - Welcome message not found! expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.example.forcegetregister.steps.RegisterSteps.loginShouldBeSuccessful(RegisterSteps.java:148)
at ✽.login should be successful(file:///C:/Users/devop/OneDrive/Masaüstü/ForgeceRegister/src/test/resources/features/register.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- org.example.forcegetregister.runners.TestRunner -->
